pandas/tests/tools/test_to_numeric.py

```diff
@@ -627,3 +627,12 @@ def test_non_coerce_uint64_conflict(errors, exp):
     else:
         result = to_numeric(ser, errors=errors)
         tm.assert_series_equal(result, ser)
+
+
+@pytest.mark.parametrize("dc1", ["integer", "float", "unsigned"])
+@pytest.mark.parametrize("dc2", ["integer", "float", "unsigned"])
+def test_downcast_empty(dc1, dc2):
+
```

Comment:
Can you add a comment that points to the original issue?

something like:

```
# https://github.com/pandas-dev/pandas/issues/32493
```

Should do it